contract/contract.go

```diff
@@ -124,6 +127,19 @@ func (j *jsonrpcTransaction) Build() error {
 	if j.to != ethgo.ZeroAddress {
 		rawTxn.To = &j.to
 	}
+
+	if j.eip1559 {
+		rawTxn.Type = ethgo.TransactionDynamicFee
+
+		// use gas price as fee data
+		gasPrice, err := j.client.GasPrice()
+		if err != nil {
+			return err
+		}
+		rawTxn.MaxFeePerGas = new(big.Int).SetUint64(gasPrice)
+		rawTxn.MaxPriorityFeePerGas = new(big.Int).SetUint64(gasPrice)
```

Comment:
Thank you for beginning EIP-1559! 
Is the intent to fully implement the `max_fee_pergas` and `max_priority_fee_pergas`?  This is inserting the legacy `gasPrice` into both of these values. So the gas refund savings will not be returned since the priority fee will consume all of the available `gasPrice`.